src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $cellGap;
+  width: $calendarWidth;
+  padding: $calendarPadding;
+
+  &__day {
+    box-sizing: border-box;
+    @extend %centering;
+    @include cellSize($cellSize);
+    background-color: #eee;
+    border: 1px solid #000;
+    cursor: pointer;
+    transition: all 0.5s;
```

Comment:
avoid using `all` - it slows down page load times
specify individual transitions

Reply:
```suggestion
    transition: transform 0.5s, background-color 0.5s;
```